### ISSUE TITLE
Toggle GUI via console command

### ISSUE THREAD

**Issue by jayden-chan** [drobilla/jalv#54]
This feature would build on #31.

It would be nice to have plugins running in headless mode being controlled by the console commands, but also have the option to toggle the GUI with a console command.

I actually "implemented" this in a private fork for the GTK3 host, but the solution was hacky and it completely broke after I pulled the most recent round of changes yesterday. I don't think I have a sufficient understanding of the codebase or DSP in general to implement this myself.

**Comment by drobilla** [comment]
> I actually "implemented" this in a private fork for the GTK3 host, but the solution was hacky and it completely broke after I pulled the most recent round of changes yesterday.

Yeah, apologies, I have been refactoring everything heavily so any changes are going to massively conflict.  All of these command feature requests are part of the rationale for why, though: the command interface is pretty toy-ish, and Jalv wasn't designed to have a bunch of parallel control systems working at once (especially remote and/or cross-UI ones).

The current architecture (or lack thereof...) can't bear the weight of a ton of hacks to pretend that a simple interface designed for twiddling parameters in the headless console app is a full-blown remote control protocol that can be used alongside a UI.  So, I'm working towards having an architecture that actually can.
